Proposals/xGov180.md

```diff
@@ -0,0 +1,61 @@
+---
+id: 180
+period: 4
+title: Research Article on Algorand Voting Technologies
+author: Brian Haney (@Bhaney44)
+email: bhaney3@alumni.nd.edu
+discussions-to: https://forum.algorand.org/t/xgov-180-research-on-algorand-voting-technologies/11744
+company_name: Independent
+category: Other
+focus_area: Other
+open_source: Yes
+funding_type: Proactive
+amount_requested: 14000
+delivery_date: 2024-08-01
+status: Final
+---
+
+## Abstract
+With 2024 being an election year in the United States, voting security has never been a hotter or more important topic. Algorand has unique features that have allowed several voting technologies to be invented, which provide an order of magnitude improvement over traditional voting machines and paper ballots. This research seeks to explain the basis for these different voting technologies and explore potential avenues for scalable adoption of Algorand in election voting. This is important because voting is a real use case, where Algorand provides a huge advantage in cost, performance, and efficiency to legacy systems.
+
+## Team
+Brian Haney
+
+## Experience with Algorand
+I have full stack OSS experience building and managing development of voting software on Algorand.
+
+<a href= "https://github.com/ChoiceCoin/v2"> Choice Coin v2 </a>
+
+<a href= "https://github.com/ChoiceCoin/Voting"> Choice Coin Voting </a>
+
+I also have extensive experience writing research on Algorand.
+
+<a href="https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3913316"> Decentralized Decisions on Algorand with Choice Coin </a>
+
+<a href="https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3819055"> Algorand Autonomous </a>
+
+I am also active in the Algorand Developer Forum and have been participating in discussion there since April of 2021. 
+
+<a href = "https://forum.algorand.org/u/bhaney44">Brian Haney Algorand Forum</a>
+
+## Present Proposal
+- Research. (2 weeks) Collecting data, articles, and information relating to voting technologies and Algorand.
+- Outling. (1 week) Outlining the articles substance and identifying particular points and topics.
+- Drafting. (2 weeks) Writing the article.
+- Finalizing. (1 week) Reviewing, revising, and publishing the article.
+
+### Deliverable
+The deliverable will be a research article of at least 7,500 words with at least 75 sources. The sources will be professional, peer-reviewed, academic, and patent publications. The article will be published in an open source database and on GitHub. 
```

Comment:
Please provide public URL of where/how the community can check proposal deliverables.

Reply:
I can post deliverables and progress here: https://github.com/Bhaney44/Algorand-Voting-Research